static/js/Autocomplete.jsx

```diff
@@ -174,31 +174,37 @@ const EntitySearchSuggestion = ({label, onClick, type, url, ...props}) => {
     );
 }
 
-const SearchInputBox = ({getInputProps, suggestions, highlightedIndex, hideHebrewKeyboard,
+const SearchInputBox = ({getInputProps, suggestions, highlightedIndex, hideHebrewKeyboard, setInputValue,
                         setSearchFocused, searchFocused,
                             submitSearch, redirectToObject}) => {
 
+    const getInputValue = () =>{
+        return otherDownShiftProps.value || document.getElementsByClassName('keyboardInput')[0].value;
```

Comment:
this function should use `getVirtualKeyboardInputValue()`

Reply:
Also, this is very dangerous since you're getting the first instance of a CSS class. If this class is used by some other element this could break. Can you come up with a more specific CSS selector?